src/App.tsx

```diff
@@ -5,30 +5,39 @@ import '@fontsource/roboto/700.css';
 
 import './App.css';
 
-import InvoiceList from './components/InvoiceList';
-import Invoice from './components/Invoice';
+import InvoiceList from './components/InvoiceList/InvoiceList';
 
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 import Layout from './components/Layout';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import React from 'react';
```

Comment:
https://epicreact.dev/importing-react-through-the-ages/ Good read about importing react :)

Reply:
To be done